sqlglot/dialects/exasol.py

```diff
@@ -19,6 +22,19 @@ def _sha2_sql(self: Exasol.Generator, expression: exp.SHA2) -> str:
     return self.func(func_name, expression.this)
 
 
+# https://docs.exasol.com/db/latest/sql_references/functions/alphabeticallistfunctions/trunc%5Bate%5D%20(datetime).htm
+# https://docs.exasol.com/db/latest/sql_references/functions/alphabeticallistfunctions/trunc%5Bate%5D%20(number).htm
+def _parse_trunc(args: t.List) -> exp.Expression:
+    if len(args) == 2:
+        first = args[0]
+        if isinstance(first, (exp.Column, exp.Date, exp.Timestamp, exp.Cast)) or (
+            isinstance(first, exp.Literal) and first.is_string
+        ):
+            return exp.DateTrunc(this=first, unit=args[1])
```

Comment:
Can you elaborate a bit on the reasoning behind this branch?

Reply:
Alright I would look into it

Reply:
@georgesittas I have updated the PR to reflect the suggestion. I also added back the test I did override mistakenly.